listenbrainz/webserver/templates/navbar.html

```diff
@@ -1,96 +1,195 @@
+                Current site status
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="{{ url_for('index.terms_of_service') }}">
+                Terms of service
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="https://metabrainz.org">
+                About MetaBrainz
+              </a>
+            </li>
           </ul>
         </li>
-    </ul>
+      </ul>
 
-    {% if not hide_navbar_user_menu %}
-      <ul class="nav navbar-nav navbar-right">
+      <form class="d-flex" role="search" method="GET" action="/search">
+        <input type="search"
+               name="search_term"
+               aria-label="Search"
+               class="form-control"
+               placeholder="Search Listener"
+               required/>
+        <button type="submit" class="btn btn-primary me-4">
+          <img src="{{ url_for('static', filename='img/icons/search-solid.svg') }}"
+               width="18px"
+               height="18px"
+               alt="Search" />
+        </button>
+      </form>
+
+      {% if not hide_navbar_user_menu %}
+      <ul class="nav navbar-nav me-2">
         {% if not current_user.is_authenticated or current_user.is_anonymous %}
-          <li>
-            <a href="{{ url_for('login.index') }}">Sign in</a>
-          </li>
+        <li class="nav-item">
+          <a class="nav-link" href="{{ url_for('login.index') }}">Sign in</a>
+        </li>
         {% else %}
-          <li class="dropdown">
-            <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-              {{ current_user.musicbrainz_id }} <span class="caret"></span>
-            </a>
-            <ul class="dropdown-menu" role="menu">
-              <li><a href="{{ url_for('profile.info') }}">Settings</a></li>
-              <li><a href="{{ url_for('user.playlists', user_name=current_user.musicbrainz_id) }}">Playlists</a></li>
-              <li><a href="{{ url_for('recommendations_cf_recording.info', user_name=current_user.musicbrainz_id) }}">Tracks you might like</a></li>
-              <li><a href="{{ url_for('profile.music_services_details') }}">Connect Services</a></li>
-              <li><a href="{{ url_for('profile.import_data') }}">Import listens</a></li>
-              <li><a href="{{ url_for('profile.export_data') }}">Export</a></li>
-              <li><a href="{{ url_for('user.missing_mb_data', user_name=current_user.musicbrainz_id) }}">Add Missing MusicBrainz Data</a></li>
-              <li><a href="{{ url_for('login.logout') }}">Sign out</a></li>
-            </ul>
-          </li>
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#">
+            {{ current_user.musicbrainz_id }} <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu" role="menu">
```

Comment:
```suggestion
          <ul class="dropdown-menu dropdown-menu-end" role="menu">
```

To ensure the menu is aligned to the right, otherwise if it is too wide i will make the page wider (and scrollable)
Here's what it looks like (with `width: max-content;` in the `.dropdown-menu` class):
<img width="366" alt="image" src="https://user-images.githubusercontent.com/6179856/169277728-08ba8aab-aaf3-4dcc-b0be-4dd2e4560789.png">

Reply:
Fixed